Projects/Foundations/Schemas/Docs-Nodes/F/Foundations/Foundations-Project/foundations-project.json

```diff
@@ -10,8 +10,8 @@
             },
             {
                 "language": "EL",
-                "text": "Η ιεραρχία Foundations Project διέπει τα περιουσιακά στοιχεία που σχετίζονται με το Superalgos. Στην τρέχουσα κατάστασή του, εξυπηρετεί το σκοπό της αναπαραγωγής ιεραρχιών.",
-                "updated": 1639090054011
+                "text": "Η ιεραρχία Foundations Project (Θεμέλια Έργου) διέπει τα στοιχεία που σχετίζονται με το Superalgos. Στην τρέχουσα κατάστασή του, εξυπηρετεί το σκοπό της αναπαραγωγής ιεραρχιών.",
+                "updated": 1641827378581
```

Comment:
You accidentally overwrite the Russian translation. Same problem in this File but absolutely fixable.

Reply:
Same as above.